service/app/controllers/BaseController.java

```diff
@@ -474,7 +474,14 @@ private String generateStackTrace(StackTraceElement[] elements) {
       builder.append(element.toString());
       builder.append("\n");
     }
-    return builder.toString();
+    String firstHundredChars = "";
```

Comment:
Create a common method that will accept String and number of char value and all logic should go in side.